opentelemetry-otlp/src/exporter/http.rs

```diff
@@ -66,12 +66,21 @@ pub struct HttpExporterBuilder {
 
 impl Default for HttpExporterBuilder {
     fn default() -> Self {
+        let mut headers: HashMap<String, String> = HashMap::new();
+        headers.insert(
+            "User-Agent".to_string(),
+            format!("OTel OTLP Exporter Rust/{}", env!("CARGO_PKG_VERSION")),
```

Comment:
nit: since we are reusing those two across multiple places. Maybe a good idea to make those static string constants

Reply:
makes sense.

Reply:
alright, because we have to format/read from environment variable, not gonna work as a `const`. However, I'll pull them into a function so we don't have to rewrite _this_ everywhere, just `defaultHeaders`.